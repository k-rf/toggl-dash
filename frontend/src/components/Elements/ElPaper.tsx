import { Paper, PaperProps } from "@mui/material";

type Props = Pick<PaperProps, "children" | "sx">;

export const ElPaper = (props: Props) => {
  return <Paper {...props} />;
};

export type { Props as ElPaperProps };
