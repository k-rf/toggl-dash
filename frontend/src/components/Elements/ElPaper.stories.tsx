import type { Meta, StoryObj } from "@storybook/react";

import { ElPaper, ElPaperProps } from "./ElPaper";

export default { component: ElPaper } as Meta<ElPaperProps>;

type Story = StoryObj<ElPaperProps>;

export const Default: Story = {
  args: {
    children: "Element Paper",
  },
};
