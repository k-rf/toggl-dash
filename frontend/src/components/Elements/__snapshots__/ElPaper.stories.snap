// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Elements/ElPaper > Default 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
>
  Element Paper
</div>
`;
